ci/verify-chefdk.bat

```diff
@@ -10,5 +10,12 @@ SET TMP=%TMP%\cheftest
 RMDIR /S /Q %TEMP%
 MKDIR %TEMP%
 
+REM ; Ensure the calling environment (disapproval look Bundler) does not
+REM ; infect our Ruby environment created by the `chef` cli.
+FOR %%E IN (_ORIGINAL_GEM_PATH, BUNDLE_BIN_PATH, BUNDLE_GEMFILE, GEM_HOME, GEM_PATH, GEM_ROOT, RUBYLIB, RUBYOPT, RUBY_ENGINE, RUBY_ROOT, RUBY_VERSION) DO SET %%E=
+
+REM ; Ensure the msys2 build dlls are not on the path
+SET PATH=C:\opscode\chefdk\bin
```

Comment:
I believe you want to append or prepend `C:\opscode\chefdk\bin` and not set just that directory to the path